lib/ReactViews/ExplorerWindow/ModalPopup.tsx

```diff
@@ -17,11 +18,10 @@ interface IProps {
 }
 
 const ModalPopup: React.FC<IProps> = props => {
-  const { t } = useTranslation();
   const [inTransition, setInTransition] = useState(false);
   const animationTimeout = useRef<ReturnType<typeof setTimeout> | null>(null);
 
-  function slideIn() {
+  /* function slideIn() {
```

Comment:
I have commented this animation out, should it run only once or every time so I can reimplement it (on main it runs only on the first open of modal)

Reply:
It should run every time isVisible changes

Reply:
finally got time to deal with it, it should work now